<launch>
  <arg name="sim" default="true" />
  <arg name="robot_ip" unless="$(arg sim)" />

  <!-- BRING THE ROBOT ONLINE -->
  <include file="$(find temoto_sia5_moveit_config)/launch/moveit_planning_execution.launch">
     <arg name="sim" value="$(arg sim)" />
     <arg name="robot_ip" value="$(arg robot_ip)" unless="$(arg sim)" />
  </include>

  <!-- BRING TEMOTO ONLINE -->
  <include file="$(find temoto)/launch/pocketsphinx.launch" />

  <node name="leap_motion" pkg="leap_motion_controller" type="leap_motion" />  
  
  <node name="griffin_powermate" pkg="griffin_powermate" type="griffin_powermate" />
  
  <node name="interpret_utterance" pkg="temoto" type="interpret_utterance" />
  <node name="human_frame_broadcaster" pkg="temoto" type="human_frame_broadcaster" />
  <node name="rviz_visual" pkg="temoto" type="rviz_visual" />

  <!-- MoveIt interface for sia5 -->
  <node name="move_robot" pkg="temoto" type="move_robot">
    <param name="movegroup" value="sia5" />
  </node>
  
  <!-- Start temoto teleoperation -->
  <node name="start_teleop" pkg="temoto" type="start_teleop" launch-prefix="xterm -title '手元 TELEOPERATOR' -e">
    <param name="primary_hand" type="str" value="left" />
    <param name="navigate" type="bool" value="false" />
    <param name="manipulate" type="bool" value="true" />
  </node>

  
</launch>

